tdrs-backend/tdpservice/data_files/util.py

```diff
@@ -42,21 +42,44 @@ def format_error_msg(x):
         ('month', lambda x: calendar.month_name[
             int(str(x['rpt_month_year'])[4:])
             ] if x['rpt_month_year'] else None),
-        ('error_type', lambda x: x['error_type']),
         ('error_message', lambda x: format_error_msg(chk(x))),
         ('item_number', lambda x: x['item_number']),
         ('item_name', lambda x: ','.join([i for i in chk(x)['fields_json']['friendly_name'].values()])),
         ('internal_variable_name', lambda x: ','.join([i for i in chk(x)['fields_json']['friendly_name'].keys()])),
         ('row_number', lambda x: x['row_number']),
-        ('column_number', lambda x: x['column_number'])
     ]
 
     # write beta banner
-    worksheet.write(row, col,
-                    "Error reporting in TDP is still in development." +
-                    "We'll be in touch when it's ready to use!" +
-                    "For now please refer to the reports you receive via email")
+    worksheet.write(
+        row, col,
+        "Please refer to the most recent versions of the coding " +
+        "instructions (linked below) when looking up items " +
+        "and allowable values during the data revision process"
+    )
+
+    row, col = 1, 0
+    worksheet.write_url(
+        row, col,
+        'https://www.acf.hhs.gov/ofa/policy-guidance/tribal-tanf-data-coding-instructions',
+        string='Tribal TANF Instructions'
```

Comment:
@reitermb @jtimpe i'm a little concerned about both sets of coding instructions being included in the report. could the following help with clarity? i'm not sure about accessibility implications associated this suggestion: 

![image](https://github.com/raft-tech/TANF-app/assets/63075587/f3f8da8f-e79b-4521-acf2-6657dfeae540)

Reply:
I don't believe this should pose any accessibility concerns!

Reply:
@jtimpe whenever you get a chance, please let us know if this is feasible. also, since this PR depends on the error guide, we'll try to get that merged in before this PR is released to prod.